packages/boot/test/bootstrapTests/orchestration.test.ts

```diff
@@ -190,6 +197,7 @@ test.serial('stakeAtom - smart wallet', async t => {
     encoding: 'bech32',
   };
 
+  // This will trigger the immediate ack of the mock bridge
```

Comment:
~~Is "This" related to `executeOffer`?~~ It seems to be due to `ackImmediately`. Please consider making this comment more explicit

Reply:
I tried to word it in a way that wouldn't be false if that code was refactored